src/AppBundle/Resources/views/Pa/Settings/profileEdit.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'AppBundle:Layouts:application.html.twig' %}
+
+{% import 'AppBundle:Macros:macros.html.twig' as macros %}
+
+{% set translationDomain = "pa-profile" %}
+{% trans_default_domain translationDomain %}
+
+{% block htmlTitle %}Settings - update your account{% endblock %}
+{% block pageTitle %}Update your account{% endblock %}
+{% block pageTitleClass %}heading-large{% endblock %}
+{% block supportTitleTop %}
+    <span class="heading-secondary">{{ 'yourDetails.supportTitle' | trans }}</span>
+{% endblock %}
+
+{% block breadcrumbs %}
+    {{ macros.breadcrumbsPASettings('profile-edit') }}
+{% endblock %}
+
+{% block pageContent %}
+    {{ form_start(form, {attr: {novalidate: 'novalidate' }}) }}
+
+    {{ form_input(form.firstname,'form.firstname') }}
+
+    {{ form_input(form.lastname,'form.lastname') }}
+
+    {{ form_input(form.email,'form.email') }}
+
+    {{ form_input(form.jobTitle, 'form.jobTitle') }}
+
+    {{ form_input(form.phoneMain,'form.phoneMain') }}
```

Comment:
same here. I see the same fields used in add, edit and profileEdit. Worth putting common fields  in the same shared template ? not sure what @Biggsen prefers